dev-packages/node-integration-tests/suites/tracing/meta-tags-twp-errors/server.js

```diff
@@ -5,13 +5,15 @@ Sentry.init({
   dsn: 'https://public@dsn.ingest.sentry.io/1337',
   transport: loggingTransport,
   beforeSend(event) {
-    event.contexts = {
-      ...event.contexts,
-      traceData: {
-        ...Sentry.getTraceData(),
-        metaTags: Sentry.getTraceMetaTags(),
-      },
-    };
+    if (!event.contexts.traceData) {
```

Comment:
why is this change necessary?

Reply:
Actually, two of the tests have become so similar that I'll just remove the beforeSend hook and one of the tests.